main.js

```diff
@@ -42,5 +44,9 @@ function displayGrid() {
 };
 
 function readySaveButton() {
-  saveButton.classList.add('.button-ready')
+  console.log("I'm in the readySaveButton func");
+  if(ideaTitle.value !== '' && ideaBodyText.value.length > 0){
+    console.log("I'm in the readySaveButton if statement");
+    saveButton.classList.add('button-ready');
+  }
```

Comment:
This is the one that we figured out we have to try and sort with (!ideaTitle.value || !ideaBodyText.value)